packages/docs/docs/contributing/bounty.md

```diff
@@ -0,0 +1,35 @@
+---
+image: /generated/articles-docs-contributing-bounty.png
+title: How to take a bounty issue
+sidebar_label: How to take a bounty issue
+crumb: Contributing
+---
+
+It’s nice to see community members contribute to Remotion’s code base. There are rewards for specific issues on GitHub to keep engagement high. These rewards are called bounties.
```

Comment:
"To keep engagement high" sounds very boomer corporate 🙈

How about the first sentence being "Some issues carry cash bounties that you can earn by solving them."